Automattic/wp-calypso
# chore(deps): Remove lodash usage from tree-select

This allows us to drop the lodash dependency entirely from tree-select and prevents us from shipping two copies of lodash in production builds. A hack for the second problem, but it works for now.

#### Testing instructions

* e2e tests should suffice. 
* This also updates a number of transitive deps (due to an `npm run update-deps` run), so unexpected things may change.

Fixes #30378
